src/utils/sformat.fsi

```diff
@@ -53,6 +53,13 @@ namespace Microsoft.FSharp.Text.StructuredFormat
         | Unbreakable
         | Breakable of int
         | Broken of int
+
+    [<NoEquality; NoComparison>]
+#if COMPILER
+    type internal BoxRange = BoxRange of obj
```

Comment:
Why are we not just passing the range around? Using obj when we know the type is a serious code smell.